### DISCUSSION TITLE
Upload provider documentation

### DISCUSSION THREAD

**Discussion by kyuumeitai** [strapi/strapi#5977] [Legacy Topics]
Hi there. Amazing job with strapi, guys!

When I upload my docker image with Strapi v3.0.0-beta.20 I was unable to upload images to my custom MinIO provider, I realised that in the /packages/strapi-provider-upload-aws-s3 changed the way provider configures, that's ok and checking the code I was able to update my provider (old fork with updated methods, if you have suggestions please give an issue! https://github.com/kyuumeitai/strapi-provider-upload-minio ), it works, but there's no documentation about it, maybe you should menction it:

- Configuring the `./extensions/upload/config/settings.json` path instead of fields (this change is awesome: most of our setups are managed via files instead of control panel)
- Adding the following in your package.json:
``` 
"strapi": {
    "isProvider": true
},
```

**Comment by lauriejim** [comment]
Hello I don't really understand your topic here.
There is a migration guide to update the config for upload providers.
